EjercicioClases/Model/User/Seller.pm

```diff
@@ -25,4 +24,10 @@ sub getSellerCode {
     return $self->{sellerCode};
 }
 
+sub save{
+    my ($self, $seller) = @_;
+    Controller::DatabaseController->save('sellers', $seller);
```

Comment:
Same as before, also maybe you can move the _save_ method into the parent class and avoid writing the method twice.